### PR TITLE
1 oct

### PR DESCRIPTION
**Pull request by MarianBastiurea**

### CODE REVIEW

File: frontend/bees-app/src/App.js [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 function App() {
   return (
     <Router>
       <div className="App">
         <Routes>
-          <Route path="/" element={<HomePage/>} />
-          <Route exact path="/another-page" component={<GameView/>}/>
-          <Route exact path="/another-page" component={<SellHoney/>}/>
-          
+          <Route path="/" element={<SellHoney />} />
```
Comment (andreiursudev) [inline frontend/bees-app/src/App.js]: Use correct route paths.

File: frontend/bees-app/src/components/FormularStart.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const FormularStart = ({ handleClose }) => {
```
Comment (andreiursudev) [inline frontend/bees-app/src/components/FormularStart.js]: Give proper name to constant.

File: frontend/bees-app/src/components/GameView.js [hunk 3]
```diff
@@ -1,138 +1,85 @@
...
 import '../App.css';
+import rapeseedFlower from '../rapeseed-flower.jpg';
+import acaciaFlower from '../acacia-flower.jpg';
+import lindenFlower from '../linden-flower.jpg';
+import falseIndigoFlower from '../false-indigo-flower.jpg'
+import sunFlower from '../sun-flower.jpg';
+import wildFlower from '../wild-flower.jpg';
+
+
+class CardGame extends Component {
```
Comment (andreiursudev) [inline frontend/bees-app/src/components/GameView.js]: CardGame is not good name. Implies is a game of cards.

File: frontend/bees-app/src/components/GameView.js [hunk 4]
```diff
@@ -1,138 +1,85 @@
...
-                                <p>rapeseed honey: 36.6kg</p>
-                                <p>Total honey: 36.6kg</p>
-                            </div>
-                        </div>
 
+const GameView = () => {
+    return (
+        <div className="body-gameView">
+            <div className="container" style={{ marginTop: '50px' }}>
+                <div className="row">
```
Comment (andreiursudev) [inline frontend/bees-app/src/components/GameView.js:34]: Row of games can be a react component.

File: frontend/bees-app/src/components/HomePage.js [hunk 5]
```diff
@@ -1,16 +1,49 @@
-import React from 'react';
+import React, { Component, useState } from 'react';
 import '../App.css';
+import FormularStart from './FormularStart';
+
+class CardGame extends Component {
```
Comment (andreiursudev) [inline frontend/bees-app/src/components/HomePage.js]: This component is duplicated.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: frontend/bees-app/src/App.js (hunk 1 above)
Comment (MarianBastiurea) [reply frontend/bees-app/src/App.js]: done

File: frontend/bees-app/src/components/FormularStart.js (hunk 2 above)
Comment (MarianBastiurea) [reply frontend/bees-app/src/components/FormularStart.js]: done

File: frontend/bees-app/src/components/GameView.js (hunk 3 above)
Comment (MarianBastiurea) [reply frontend/bees-app/src/components/GameView.js]: change it in HiveCard

File: frontend/bees-app/src/components/FormularStart.js (hunk 2 above)
Comment (MarianBastiurea) [reply frontend/bees-app/src/components/FormularStart.js]: Change it in LoginComponents

File: frontend/bees-app/src/components/HomePage.js (hunk 5 above)
Comment (MarianBastiurea) [reply frontend/bees-app/src/components/HomePage.js]: change it to ApiaryCard

File: frontend/bees-app/src/components/HomePage.js [hunk 6]
```diff
@@ -23,214 +51,41 @@ const HomePage = () => {
                     </li>
                 </ul>
             </div>
-
             <div className="row pt-3">
```
Comment (andreiursudev) [inline frontend/bees-app/src/components/HomePage.js]: Row of games can be a react component.

Comment (MarianBastiurea) [reply frontend/bees-app/src/components/HomePage.js]: done. create ApiaryCardRow

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**